### PR TITLE
[CSL-2060] Utilize UI autocomplete in React example

### PR DESCRIPTION
**Pull request by esezen**
After some discussion with Jesse, changed the product count from 6 to 8 to use the defaults of OS UI and show simplicity of implementation.

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -10,6 +10,138 @@
   max-height: 660px;
 }
 
+div.cio-autocomplete .cio-submit-btn, .cio-clear-btn {
```
Comment (esezen) [inline src/index.css:13]: How do we feel about the specificity of these?

**Review by mocca102 — APPROVED** [review]
LGTM!

File: src/index.css (hunk 1 above)
Comment (esezen) [reply src/index.css:13]: I was using the default styles at first and I needed the element selectors to make it specific. Now that I'm not using the default styles, I think they are unnecessary. I'll update

**Review by mocca102 — APPROVED** [review]
LGTM!

**Pull request merged**